cmake/platform_android.cmake

```diff
@@ -70,6 +70,18 @@ function(setup_litecore_build)
     )
 endfunction()
 
+function(setup_support_build)
+    setup_support_build_linux()
+    target_compile_definitions(
+        Support PRIVATE
+        -DLITECORE_USES_ICU=1
```

Comment:
If this is needed in Android then it's needed in Linux as well, so better to move this compile definition into `platform_linux` (which both Linux and Android derive from)